src/libs/actions/PaymentMethods.ts

```diff
@@ -82,19 +96,19 @@ function getMakeDefaultPaymentOnyxData(bankAccountID, fundID, previousPaymentMet
         onyxData[0].value.errors = null;
     }
 
-    if (previousPaymentMethod) {
+    if (previousPaymentMethod?.methodID) {
         onyxData.push({
             onyxMethod: Onyx.METHOD.MERGE,
             key: previousPaymentMethod.accountType === CONST.PAYMENT_METHODS.BANK_ACCOUNT ? ONYXKEYS.BANK_ACCOUNT_LIST : ONYXKEYS.FUND_LIST,
             value: {
-                [previousPaymentMethod.methodID]: {
+                [previousPaymentMethod?.methodID]: {
```

Comment:
@JKobrynski Do you really need `?.` here, you already checked it in the if condition above

Reply:
Missed it, thanks!